src/qinst/serial_inst.py

```diff
@@ -41,10 +41,12 @@ def disconnect(self):
         if self.serial.is_open:
             self.serial.close()
 
-    def write(self, cmd):
+    def write(self, cmd, sleep=False):
         """Write a message to the serial port."""
         if self.serial is not None:
```

Comment:
So che non l'hai cambiato tu, ma qui (e nel resto del file) dovrebbe essere:
```suggestion
        if self.serial.is_open:
```
Coglierei l'occasione per sistemarlo